readme-vars.yml

```diff
@@ -6,6 +6,7 @@ project_url: "https://github.com/plankanban/planka/"
 project_logo: "https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/planka-logo.png"
 project_blurb: "[{{ project_name|capitalize }}]({{ project_url }}) is an elegant open source project tracking tool."
 project_lsio_github_repo_url: "https://github.com/linuxserver/docker-{{ project_name }}"
+project_categories: "Content Management"
```

Comment:
Content Management doesn't really fit but I'm not sure what's best. Project Mangement is broad but I'm not sure we've got anything else that fits other than maybe Kimai.

Perhaps have a Business category and include this, Kimai, maybe Netbox? Dunno.

Reply:
Added business